### ISSUE TITLE
Issue in generating code

### ISSUE THREAD

**Issue by alishari** [comahe-de/i18n4k#7]
I have faced an issue in [SpotiFlyer](https://github.com/Shabinder/SpotiFlyer) project. Some strings without parameters are detected as parameterized strings.

For example, for "What Went Wrong...", the generated code must be getLocalizedStringFactory0, but it is getLocalizedStringFactory1.

The problem is solved by passing an empty string. 

**Problems:**

```
  /**
   * Copy to Clipboard
   */
  @JvmStatic
  public val copyToClipboard: LocalizedStringFactory1 = getLocalizedStringFactory1(15)

  /**
   * Copy Paste Below Code while creating Github Issue / Reporting this issue for better help.
   */
  @JvmStatic
  public val copyCodeInGithubIssue: LocalizedStringFactory1 = getLocalizedStringFactory1(14)

  /**
   * What Went Wrong...
   */
  @JvmStatic
  public val whatWentWrong: LocalizedStringFactory1 = getLocalizedStringFactory1(83)
```
  
  
  **Versions:**
* kotlinVersion = "1.5.10"
* i18n4kVersion="0.1.2"
* Android Studio="Android Studio Arctic Fox | 2020.3.1"
* 

Generated code: https://justpaste.it/1wu1i

**Comment by comahe-de** [comment]
Hi, @alishari,
I found the bug: it happens when a key has missing translation. In the file "Strings_pt_BR.properties" the key "whatWentWrong" is missing. 
I will fix in in the near future....

**Comment by comahe-de** [comment]
Fixed the problem in commit 7477d006fc559f0b20e2883f2ed8d1984e4007d3. 

A new release will be published in the next days.
